graylog2-web-interface/src/views/components/common/ActionDropdown.jsx

```diff
@@ -97,22 +97,41 @@ class ActionDropdown extends React.Component<ActionDropdownProps, ActionDropdown
     this.setState(({ show }) => ({ show: !show }));
   };
 
-  render() {
-    const { children, container, element } = this.props;
-    const { show } = this.state;
+  adjustOptionOnSelectProp = (option: React.Element<*>, updateDepth: number) => {
```

Comment:
please rename `option` to `child` or `node`, it is in no case an option and it took me 3 minutes to understand that

Reply:
Thanks for having a look. I've adjusted the naming.